_posts/2020-08-30-commenting-system-blog.md

```diff
@@ -93,9 +93,12 @@ The below is the template code used to create all the resources needed
     }
     }
 ```
+The above template creates from the command `dotnet new -i Amazon.Lambda.Templates` and what it creates is a serverless function, API Gateway and DynamoDB table.
+Lambda (serverless) function is triggered by API Gateway which in turn does CRUD operations in DynamoDB.
```

Comment:
does or performs?

Reply:
You are right. Thanks for pointing that out